### DISCUSSION TITLE
Journal des variables - Acquit All

### DISCUSSION THREAD

**Discussion by WIToliv** [witsa/synapps#625] [Ideas]
Hello, 
Avoir la possibilité dans le journal des variables d'avoir un bouton intégré pour acquitter l'ensemble des alarmes d'un coup comme sur le journal du REDY.

![image](https://github.com/witsa/synapps/assets/101122670/ecc37ee7-a5a7-431e-bc11-abdfd49a6c86)

![image](https://github.com/witsa/synapps/assets/101122670/7aad1a62-edbd-4cfc-b7ac-d902c930b645)

Merci

**Comment by Raphwit** [comment]
Hello, on note l'amélioration pour une future mise à jour sera ajoutée, merci !

**Comment by Raphael-zoreille** [comment]
Bonjour, 

Si vous avez possibilité d'ajouter tout ce qu'on trouver dans les colonnes pour les synoptiques, ça serait super. 
Là j'ai besoin d'afficher la colonne "Zone" et "Ack par", mais elles n'y sont pas. 

Et petite personnalisation supplémentaire si possible, pouvoir fixer la largeur de chaque colonne. 
Merci
Cordialement
